rascal-vscode-extension/src/RascalExtension.ts

```diff
@@ -38,14 +38,16 @@ import { RascalLibraryProvider } from './ux/LibraryNavigator';
 import { FileType } from 'vscode';
 import { RascalDebugViewProvider } from './dap/RascalDebugView';
 
+export const extensionLog: vscode.LogOutputChannel = vscode.window.createOutputChannel("Rascal Extension", {log: true});
```

Comment:
I don't think it should be this global, it should be owned by the `RascalExtension` class and disposed along side of it. Or maybe we need to have this at the root of the extension? and pass it in as a parameter to the constructor?

Roughly: I don't want our extension to register output channels if we're not active. And I want to be able to survive package updates without dangling windows etc.